scripts/events_module/short/short_event.py

```diff
@@ -69,22 +102,34 @@ def __init__(
         if self.r_c:
             if "age" not in self.r_c:
                 self.r_c["age"] = ["any"]
+            elif "any" not in self.r_c["age"]:
+                self.weight += self.num_of_ages - len(self.r_c["age"])
             if "status" not in self.r_c:
                 self.r_c["status"] = ["any"]
             if "relationship_status" not in self.r_c:
                 self.r_c["relationship_status"] = []
+            else:
+                self.weight += len(self.r_c["relationship_status"])
             if "skill" not in self.r_c:
                 self.r_c["skill"] = []
+            else:
+                self.weight += self.num_of_skills - len(self.r_c["skill"])
             if "not_skill" not in self.r_c:
                 self.r_c["not_skill"] = []
+            else:
+                self.weight += len(self.r_c["not_skill"])
             if "trait" not in self.r_c:
                 self.r_c["trait"] = []
+            else:
+                self.weight += self.num_of_traits - len(self.r_c["trait"])
             if "not_trait" not in self.r_c:
                 self.r_c["not_trait"] = []
-            if "age" not in self.r_c:
-                self.r_c["age"] = []
+            else:
+                self.weight += len(self.r_c["not_trait"])
             if "backstory" not in self.r_c:
                 self.r_c["backstory"] = []
+            else:
+                self.weight += 1
```

Comment:
if we have both `if not a` and `else` checks, can we get these swapped around so that it's `if a` and `else`? just for logical flow.